contrib/testgen/gen_key_io_test_vectors.py

```
@@ -1,20 +1,24 @@
-# Released under MIT License
-import os
+
 from itertools import islice
-from base58 import b58encode_chk, b58decode_chk, b58chars
+import os
 import random
+import sys
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '../../test/functional'))
+
+from test_framework.address import base58_to_byte, byte_to_base58, b58chars  # noqa: E402
```

Ruff (0.12.2)

Unused `noqa` directive (unused: `E402`)

Remove unused `noqa` directive

(RUF100)